helm/kfp-operator/templates/eventing/status-updater.yaml

```diff
@@ -84,6 +84,6 @@ spec:
         parameters:
         - src:
             dependencyName: events
-            dataKey: body.data
+            dataKey: body
```

Comment:
hmmm think this might cause clients issues, might need to discuss this.

Reply:
good point, I've changed the code to retain the data key when published to nats so existing functionality should match what is currently used